angr/analyses/cfg/cfg_fast_soot.py

```diff
@@ -118,15 +118,15 @@ def _generate_cfgnode(self, cfg_job, current_function_addr):
 
         try:
 
-            if addr in self._nodes:
-                cfg_node = self._nodes[addr]
+            if addr in self.model.nodes():
+                cfg_node = self.model.nodes()[addr]
                 soot_block = cfg_node.soot_block
```

Comment:
Wrong behaviour? for `self.model.nodes()` returns "an iterator of all nodes in the graph".

Reply:
Good catch!